lib/logstash/inputs/twitter.rb

```diff
@@ -46,7 +46,13 @@ class LogStash::Inputs::Twitter < LogStash::Inputs::Base
   config :oauth_token_secret, :validate => :password, :required => true
 
   # Any keywords to track in the twitter stream
-  config :keywords, :validate => :array, :required => true
+  config :keywords, :validate => :array, :required => false
+
+  # Any locations to track in the twitter stream
+  config :locations, :validate => :array, :required => false
+
+  # Any users to follow in the twitter stream
```

Comment:
It may be worth documenting that keywords+locations+follows are additive. Initially, I was almost expecting using both keywords and locations together would result in tweets satisfying both the keyword and location filters, but per the [Twitter gem docs](http://www.rubydoc.info/gems/twitter/Twitter/Streaming/Client#filter-instance_method), using keywords and location together acts as an 'or' (matching either the keywords *or* the locations, not requiring both)